internal/server/passwordreset.go

```diff
@@ -23,16 +24,11 @@ func (a *API) RequestPasswordReset(c *gin.Context, r *api.PasswordResetRequest)
 		return nil, err
 	}
 
-	// TODO: adjust domain for orgs when here
-	link, _ := url.Parse(email.AppDomain)
-	link.Path = "/password-reset"
-	query := link.Query()
-	query.Add("token", token)
-	link.RawQuery = query.Encode()
+	org := data.MustGetOrgFromContext(c)
```

Comment:
This isn't set in middleware?

Reply:
That was confusing, my bad. I was thinking this was setting the org on the db, but this looks fine 👍